File: sem5/ON/L1/ex2.jl
# Jakub Jaśków
floatTypes = [Float16, Float32, Float64]
println("Zadanie 2")
for type in floatTypes
    println('\n', type, "(3(4/3-1) - 1) = ", type(3.0*type(type(4.0 / 3.0) - 1.0) - 1))
    println("macheps(", type, ") = ", eps(type))
end

File: electives/PWL/L1/ex2.pl
% Jakub Jaśków 268416
on(1, 2).
on(2, 3).
on(3, 5).
on(5, 6).

above(X, Y) :-
	on(X,Y).
above(X, Y) :-
	on(X, Z),
	above(Z, Y).
